app/_locales/zh_TW/messages.json

```diff
@@ -0,0 +1,864 @@
+  },
+  "passwordsDontMatch": {
+    "message": "您所輸入的密碼不一致"
+  },
+  "etherscanView": {
+    "message": "在 Etherscan 上查看帳戶"
+  },
+  "exchangeRate": {
+    "message": "匯率"
+  },
+  "exportPrivateKey": {
+    "message": "導出私鑰"
+  },
+  "exportPrivateKeyWarning": {
+    "message": "您需要自行負擔導出私鑰產生的風險"
+  },
+  "failed": {
+    "message": "失败"
+  },
+  "fiat": {
+    "message": "FIAT",
+    "description": "Exchange type"
+  },
+  "fileImportFail": {
+    "message": "檔案導入失敗？點擊這裡！",
+    "description": "Helps user import their account from a JSON file"
+  },
+  "from": {
+    "message": "來源地址"
+  },
+  "fromToSame": {
+    "message": "來源和目的地址不能一樣"
+  },
+  "fromShapeShift": {
+    "message": "來自 ShapeShift"
+  },
+  "gas": {
+    "message": "Gas",
+    "description": "Short indication of gas cost"
+  },
+  "gasFee": {
+    "message": "Gas 費用"
+  },
+  "gasLimit": {
+    "message": "Gas 上限"
+  },
+  "gasLimitCalculation": {
+    "message": "我們根據網路成功率算出建議的 Gas 上限。"
+  },
+  "gasLimitRequired": {
+    "message": "必需填寫 Gas 上限"
+  },
+  "gasLimitTooLow": {
+    "message": "Gas 上限至少為 21000"
+  },
+  "gasPrice": {
+    "message": "Gas 價格 (GWEI)"
+  },
+  "gasPriceCalculation": {
+    "message": "我們根據網路成功率算出建議的 Gas 價格"
+  },
+  "gasPriceRequired": {
+    "message": "必需填寫 Gas 價格"
+  },
+  "getEther": {
+    "message": "取得 Ether"
+  },
+  "getEtherFromFaucet": {
+    "message": "從水管取得$1 Ether",
+    "description": "Displays network name for Ether faucet"
+  },
+  "greaterThanMin": {
+    "message": "必須要大於等於 $1。",
+    "description": "helper for inputting hex as decimal input"
+  },
+  "here": {
+    "message": "這裡",
+    "description": "as in -click here- for more information (goes with troubleTokenBalances)"
+  },
+  "hereList": {
+    "message": "Here's a list!!!!"
+  },
+  "hide": {
+    "message": "隱藏"
+  },
+  "hideToken": {
+    "message": "隱藏代幣"
+  },
+  "hideTokenPrompt": {
+    "message": "隱藏代幣？"
+  },
+  "howToDeposit": {
+    "message": "你想怎麼存入 Ether？"
+  },
+  "holdEther": {
+    "message": "Metamask 讓您能保存 ether 和代幣, 並成為您接觸分散式應用程式的途徑."
+  },
+  "import": {
+    "message": "導入",
+    "description": "Button to import an account from a selected file"
+  },
+  "importAccount": {
+    "message": "導入帳戶"
+  },
+  "importAnAccount": {
+    "message": "導入一個帳戶"
+  },
+  "importDen": {
+    "message": "導入現成的 DEN"
+  },
+  "imported": {
+    "message": "已導入私鑰",
+    "description": "status showing that an account has been fully loaded into the keyring"
+  },
+  "infoHelp": {
+    "message": "說明 & 資訊"
+  },
+  "insufficientFunds": {
+    "message": "資金不足."
+  },
+  "insufficientTokens": {
+    "message": "代幣不足."
+  },
+  "invalidAddress": {
+    "message": "錯誤的地址"
+  },
+  "invalidAddressRecipient": {
+    "message": "接收地址錯誤"
+  },
+  "invalidGasParams": {
+    "message": "Gas 參數錯誤"
+  },
+  "invalidInput": {
+    "message": "輸入錯誤。"
+  },
+  "invalidRequest": {
+    "message": "無效的請求"
+  },
+  "invalidRPC": {
+    "message": "無效的 RPC URI"
```

Comment:
'URL'

Reply:
`URI` is the right term here https://github.com/MetaMask/metamask-extension/blob/master/app/_locales/en/messages.json#L409